avro/src/main/scala/magnolify/avro/AvroType.scala

```diff
@@ -196,14 +196,14 @@ object AvroField {
   implicit val afLong: AvroField[Long] = id[Long](Schema.Type.LONG)
   implicit val afFloat: AvroField[Float] = id[Float](Schema.Type.FLOAT)
   implicit val afDouble: AvroField[Double] = id[Double](Schema.Type.DOUBLE)
-  implicit val afBytes: AvroField[Array[Byte]] = new Aux[Array[Byte], ByteBuffer, ByteBuffer] {
+  implicit val afByteBuffer: AvroField[ByteBuffer] = new Aux[ByteBuffer, ByteBuffer, ByteBuffer] {
```

Comment:
looks like it could be declared as `id[ByteBuffer](Schema.Type. BYTES)`, but `id` needs to have an override to accommodate `d.array()`